api-reference/10 UI Widgets/GridBase/1 Configuration/onToolbarPreparing.md

```diff
@@ -190,6 +211,8 @@ The following code shows how you can customize the toolbar using this function.
 
 ---
 
+[important]If you use [DevExtreme modules](/concepts/Common/Modularity '/Documentation/Guide/Common/Modularity/'), import the editor's module when you add items other than [Button](/Documentation/ApiReference/UI_Widgets/dxButton/) or [TextBox](/Documentation/ApiReference/UI_Widgets/dxTextBox/). 
```

Comment:
It's unclear what "the editor" is because the article doesn't use term at all. The toolbar has "items". This note should rewritten so as to make clear what's editors, what's items and why the Button and TextBox doesn't require an import.